e2e_tests/tests/per_provider/normal_tests/import_key.rs

```diff
@@ -74,18 +89,33 @@ fn example_modulus_1024() -> Vec<u8> {
     ]
 }
 
+#[cfg(not(feature = "cryptoauthlib-provider"))]
 #[test]
-fn import_key() -> Result<()> {
+fn import_rsa_key() -> Result<()> {
     let mut client = TestClient::new();
     let key_name = String::from("import_key");
     if !client.is_operation_supported(Opcode::PsaImportKey) {
         return Ok(());
     }
+
     client.import_rsa_public_key(key_name, KEY_DATA.to_vec())
 }
 
+#[cfg(feature = "cryptoauthlib-provider")]
```

Comment:
Could you please also execute this test on the Mbed Crypto provider to make sure the test data is correct with the PSA specs?
```rust
#[cfg(any(feature = "mbed-crypto-provider", feature = "cryptoauthlib-provider"))]
```

Reply:
Done